src/bridge/events.rs

```diff
@@ -380,7 +380,10 @@ fn parse_map(map_value: Value) -> Result<Vec<(Value, Value)>> {
 }
 
 fn parse_string(string_value: Value) -> Result<String> {
-    string_value.try_into().map_err(ParseError::String)
+    match string_value {
+        Value::String(s) => Ok(s.into_str().unwrap_or_else(|| String::from("?"))),
```

Comment:
Looks good to me. Maybe using `\u{FFFD}` would be more appropiate though, as `?` is a pretty common character.

Reply:
Sure, sounds good. I was thinking about somehow indicating the length of `s.into_bytes` in the placeholder, but maybe that's not important.